api/suggestion/controllers/suggestion.js

```diff
@@ -1,8 +1,35 @@
 'use strict';
+const { parseMultipartData, sanitizeEntity } = require("strapi-utils");
 
 /**
  * Read the documentation (https://strapi.io/documentation/developer-docs/latest/development/backend-customization.html#core-controllers)
  * to customize this controller
  */
 
-module.exports = {};
+module.exports = {
+  /**
+   * Endpoint for creating new comments. This saves the comments in draft state,
+   * so that they can be later published by admins.
+   */
+   async comment(ctx) {
+    const { id } = ctx.params;
+
+    let entity, data;
+
+    if (ctx.is("multipart")) {
+      ({ data } = parseMultipartData(ctx));
+    } else {
+      data = ctx.request.body;
+    }
+
+    // Set published_at null so the entry will be created in draft state
+    data.published_at = null;
+    data.suggestion = id;
```

Comment:
Probably should check that the suggestion exists and if not respond with an error.

Reply:
Good point ☝️. With a bad id it seems to just link it to a non-existing entry, but not throw any errors.